articles/quickstart/native/react-native/_includes/_login.md

```diff
@@ -76,6 +88,10 @@ You can then register the identifier by adding the following snippet:
 ```
 
 ::: note
+The value org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier) is the default for apps created with React Native CLI, you may have a different value.
+:::
+
+::: note
 For further reference on linking, check [the official documentation](https://facebook.github.io/react-native/docs/linking.html)
```

Comment:
This note should be in the "Link the native module" section. I'd also add a sentence on that section telling that "libraries like this depend on other native platform's libraries such as Android and iOS, which requires us to first link those dependencies to be able to use the native code". Or something like that. It feels so lonely that section with just a header and a snippet... And then you go straight to the "how to login using auth0", which IMO should be the first thing in Configuration or at least have a separate header?

Reply:
The https://facebook.github.io/react-native/docs/linking.html isn't anything to do with native module linking. It's a section about handling universal links etc
I might just remove it from the QS, it's not something the avg person cares about anyway, it's still in the README though for more detail.

The `link the native module` is about linking the module itself into the react-native project.

I'll check other QS formatting regarding the header.